Bio/Entrez/__init__.py

```diff
@@ -113,9 +113,9 @@
 import warnings
 
 # Importing these functions with leading underscore as not intended for reuse
-from Bio._py3k import urlopen as _urlopen
```

Comment:
Bio.Entrez has already been fixed

Reply:
It hadn't at the time I rebased this - but that explains the new merge conflict.

Reply:
Rebasing...